host/tests/dsClasses/dsAudio_test_suite.yml

```diff
@@ -3,15 +3,180 @@
 dsAudio:  # Prefix must always exist
     description: "dsAudio Device Settings testing profile / menu system for UT"
     test:
-        execute: "" #Execute command
+        execute: "<PATH on Device>run.sh -p <PATH on Device>Sink_AudioSettings.yaml" #Execute command
```

Comment:
this should be a list of assets that get downloaded to the box in a specific location, just like the other asset downloads.
And the location of which is relative to the `host/tests/dsClasses/dsAudio_test_suite.yml` of where the built binaries can be found.
In order to run the testing suite today, you need to build the testing suite first, then you have the assets which can be downloaded and also executed on the box.

`<PATH on Device>run.sh -p <PATH on Device>Sink_AudioSettings.yaml"` is not a valid line, because /tmp/dsAudio/run.sh /tmp/dsAudio/testSettings.yaml" would be required.

The test settings, are all the tests menus for the suite, nothing to do with the sink or source, which is not relevant to the suite.